### PR TITLE
Added component "MatLink". 

### PR DESCRIPTION
**Pull request by PeteJobi**
**MatLink** is a link component styled exactly like a `MatButton`.

The `MatButton` component already has a `Link` parameter but it is in fact a _button_ and doesn't have the features of a _link_ like being able to open in a new tab, copying the link address e.t.c....
The styles are based on mdc-button styles, but extra styles are added for a class "mdc-link"
It doesn't have `ForceReload`, `Name`, `Value`, `Label` and `Type` params which are present in `MatButton`, and `Link` param was renamed to `Href`. Every other parameter from `MatButton` is present.

I can update the docs with the component if requested.

### CODE REVIEW

File: src/MatBlazor/Components/MatLink/BaseMatLink.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                .Add("mdc-button")
+                .Add("mdc-link")
+                .If("mdc-button--raised", () => this.Raised)
+                .If("mdc-button--unelevated", () => this.Unelevated)
+                .If("mdc-button--outlined", () => this.Outlined)
+                .If("mdc-button--dense", () => this.Dense);
+        }
+
+        /// <summary>
+        ///  Event occurs when the user clicks on an element.
```
Comment (Christian-Oleson) [inline src/MatBlazor/Components/MatLink/BaseMatLink.cs:28]: Not a blocker, but the code comments are inconsistent (e.g. here we have more spaces than in other comments.

File: src/MatBlazor/Components/MatLink/BaseMatLink.cs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        /// Stop propagation of the OnClick event
+        /// </summary>
+        [Parameter]
+        public bool OnClickStopPropagation { get; set; }
+
+        [Parameter]
+        public ICommand Command { get; set; }
+
+        /// <summary>
+        ///  Command parameter.
```
Comment (Christian-Oleson) [inline src/MatBlazor/Components/MatLink/BaseMatLink.cs:43]: Same here

File: src/MatBlazor/wwwroot/dist/matBlazor.css [hunk 3]
```diff
@@ -1,3 +1,3 @@
 @import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500);
```
Comment (Christian-Oleson) [inline src/MatBlazor/wwwroot/dist/matBlazor.css:1]: Not for this PR, but eventually, we need to move the .css files to a readable format, and use build steps to minify these on output.

File: src/MatBlazor/Components/MatLink/BaseMatLink.cs (hunk 1 above)
Comment (PeteJobi) [reply src/MatBlazor/Components/MatLink/BaseMatLink.cs]: Fixed. (I believe you're referring to the space before ```Event```. Please clarify if it's something else).

File: src/MatBlazor/Components/MatLink/BaseMatLink.cs (hunk 2 above)
Comment (PeteJobi) [reply src/MatBlazor/Components/MatLink/BaseMatLink.cs]: I have removed the space before ```Command```. Please let me know if you meant something else.

File: src/MatBlazor/wwwroot/dist/matBlazor.css (hunk 3 above)
Comment (PeteJobi) [reply src/MatBlazor/wwwroot/dist/matBlazor.css:1]: That would be nice. Editing minified css is a pain.

**Comment by enkodellc** [comment]
@PeteJobi  please review "Contributing Tips" on the Readme. https://github.com/SamProf/MatBlazor#contributing-tips  as well as this for CSS changes:  https://github.com/SamProf/MatBlazor/wiki/How-to-Edit-the-css---scss-styles-for-MatBlazor

**Pull request closed without merging**